canonical-web-and-design/maas-ui
# Revert Images page to checkboxes, but add warning for unsupported arches.

## Done
- Reverted images page to use checkboxes instead of radio buttons, but added some functionality for highlighting unsupported architectures.

## QA
- Go to the images page and check that the radio buttons are now checkboxes.
- Select 20.04 LTS and check that the i386 checkbox is disabled.
- Select more than one release and check that the i386 checkbox is enabled with a warning.
- Select i386 and check that the table of images does not include 20.04/i386
- Leave i386 checked, and uncheck all non-20.04 releases and see that the i386 checkbox becomes disabled and unchecked automatically.
- Uncheck 20.04 and check two other releases and see that the i386 checkbox is enabled with no warning
- Check that you can still download/remove images successfully. 

## Fixes
Fixes #1241 

## Screeshots
### Multiple releases selected, including 20.04
![0 0 0 0_8400_MAAS_l_images (3)](https://user-images.githubusercontent.com/25733845/84457027-ef711100-aca4-11ea-8250-d340cfd855ee.png)


### Multiple releases selected, not including 20.04
![0 0 0 0_8400_MAAS_l_images (5)](https://user-images.githubusercontent.com/25733845/84457560-5e029e80-aca6-11ea-9e54-581d4b78aba8.png)


### Just 20.04 selected
![0 0 0 0_8400_MAAS_l_images (4)](https://user-images.githubusercontent.com/25733845/84457031-f39d2e80-aca4-11ea-88c6-9b9285275c9a.png)
